Sources/Commands/Workspace.swift

```diff
@@ -269,6 +271,71 @@ public class Workspace {
         return path
     }
 
+    /// Updates the current dependencies.
+    public func updateDependencies() throws {
+        let (rootManifest, _) = try loadDependencyManifests()
+        // Only create constraints based on root manifest for the update resolution.
+        let updateConstraints = computePackageConstraints(rootManifest: rootManifest, dependencies: [])
+        // Resolve the dependencies.
+        let updateResults = try resolveDependencies(constraints: updateConstraints)
+
+        for (specifier, version) in updateResults {
```

Comment:
To go even further, would it be worth writing this so that a hypothetical client could get a preflight of what would need to happen without actually doing the update (perhaps using a dry-run flag, which would mean that the client would potentially call the API twice: first as a dry run, and then if the user approved, as an actual update operation).